builder/files/genesis-amoy.json

```diff
@@ -39,6 +39,9 @@
         },
         "validatorContract": "0x0000000000000000000000000000000000001000",
         "stateReceiverContract": "0x0000000000000000000000000000000000001001",
+        "overrideStateSyncRecordsInRange": [
+          { "startBlock": 24184305, "endBlock": 23220127, "value": 0 }
```

Comment:
How can startBlock be > endBlock?

Reply:
Sorry about the confusion. Yes, you are totally right about this. I didn't think of the case where state sync events will get filled even before 23220015.